README.md

```diff
@@ -1,23 +1,22 @@
 # Virtual_ship_classroom
 Emma's work for the MSc student material
-Please contact me at e.e.daniels1@uu.nl with any questions. 
+Please contact me at e.e.daniels1@uu.nl with any questions.
```

Comment:
Can this be more professionally worded at the top? It's a team effort now. And move contact details to the end of the readme? More in line with other repos?

Reply:
Will sit down with @ammedd to write a new README (I think this one is quite outdated)